ads/yieldbot.js

```diff
@@ -31,7 +31,7 @@ export function yieldbot(global, data) {
 
   global.ybotq = global.ybotq || [];
 
-  loadScript(global, 'https://cdn.yldbt.com/js/yieldbot.intent.js', () => {
+  loadScript(global, 'https://cdn.yldbt.com/js/yieldbot.intent.amp.js', () => {
```

Comment:
At this current moment yes, but when this merges I'll update and the script will be minified.